src/ontology/nbo-edit.owl

```diff
@@ -1264,16 +1453,11 @@ AnnotationAssertion(rdfs:label obo:NBO_0000013 "locomotory behavior")
 SubClassOf(obo:NBO_0000013 obo:GO_0040011)
 SubClassOf(obo:NBO_0000013 obo:NBO_0000002)
 
-# Class: obo:NBO_0000014 (mating behavior)
+# Class: obo:NBO_0000014 (sexual activity)
 
-AnnotationAssertion(Annotation(oboInOwl:source "NBO:GVG") obo:IAO_0000115 obo:NBO_0000014 "Behavior related to the interactions between organisms for the purpose of mating.")
-AnnotationAssertion(dc:date obo:NBO_0000014 "2011-04-14T01:03:39Z")
-AnnotationAssertion(oboInOwl:created_by obo:NBO_0000014 "George Gkoutos")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000014 "mating behaviour")
-AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0000014 "behavior_ontology")
-AnnotationAssertion(oboInOwl:xref obo:NBO_0000014 "GO:0007617")
-AnnotationAssertion(rdfs:label obo:NBO_0000014 "mating behavior")
-SubClassOf(obo:NBO_0000014 obo:NBO_0000010)
+SubClassOf(obo:NBO_0000014 obo:NBO_0000338)
+SubClassOf(obo:NBO_0000014 DataSomeValuesFrom(obo:BFO_0000050 obo:NBO_0000010))
```

Comment:
Big mistake, why "DataSomeValuesFrom" - should be `Object`

Reply:
The ROBOT template header line 2 for this column is 
**SC BFO:0000050 some % SPLIT=|**
The annotation name (line 1) is **part_of**
I don't know the workings which generate `SubClassOf(obo:NBO_0000014 DataSomeValuesFrom(obo:BFO_0000050 obo:NBO_0000010))`; presumably the ROBOT line 2 entry should be something else?

Reply:
This seems to have been a template bug.